src/rosdistro/rosdistro.py

```diff
@@ -222,7 +224,22 @@ def _fetch_package_xml(self, rosdistro):
             self._release_tags[rosdistro] = release_tag
             return package_xml, release_tag
         else:
-            raise Exception("Non-github repositories are net yet supported by the rosdistro tool")
+            release_tag = 'release/{0}/{1}/{2}'.format(rosdistro, self.name, repo.version)            
+            for nongithub in supported_non_githubs:
```

Comment:
So, the loop isn't useful yet, but upon the addition of other potential repo types, the structure makes more sense.